st2common/st2common/models/api/action.py

```diff
@@ -497,6 +497,27 @@ class ActionAliasAPI(BaseAPI, APIUIDMixin):
                 "type": "array",
                 "items": {"type": "string"},
                 "description": "Possible parameter format."
+            },
+            "aliases": {
```

Comment:
Yeah, now that I read the original explanation again I was how is this different then existing formats attribute.

Anyway, I do like this new proposed formats change though.